### DISCUSSION TITLE
Help, I think my installation is Panicking, it won't complete

### DISCUSSION THREAD

**Discussion by damaini** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#142] [Q&A] [answered]
Hello everyone.  Thanks for an excellent work.

I recently damaged my previous install, when I updated a program and my Hackintosh started crashing.  I decided to do clean new install of MacOS Catalina.  
But for the life of me, I can't get it to work.  My hunch is that it is having a Panic and it is restarting.  The reason I say this is that once I start the install, in the middle of it all of the sudden it restarts and the AFPS partitions for the booting drive are not completed. (I should have two, one with the volume name and another with the volume name data.  I only have one).  
I have tried several times, and every other 4 tries, I get the volume name data to appear, but it won't boot.
Does anyone have any idea?
Please have patience, I'm a newbie at this.

My hardware is 
Lenovo ThinkPad T460s Type 20F9.  
Procesor is an Intel Core i7 vPro model: 6600U 2.60Ghz
UEFI BIOS Version 1.54, BIOS Date 2021-12-06
Memory: 20GB
SSD 500GB
For WiFi and Bluetooth : macOS PC WiFi Card BCM94352Z M.2 NGFF 802.11a/b/g/n/ac 2.4GHz 5GHz BT4.0 

The only changes I did to the original laptop was: 
1. A bigger hard drive (but I did this from the beginning)
2. Changed the WiFi/Bluetooth for one for macOS and 100% compatible (also from the beginning, never gave me issues and the wifi works during install)
3. More memory (also from the beginning)

The only thing I can think of is the update to the BIOS I did recently to 1.54.
And YES, I did double check that I had the correct BIOS settings as specified in the install.

Thanks for your help.

**Comment by simprecicchiani** [comment, marked as answer]
Hi there, have you tried installing with a previous EFI release? I remember this kind of error popping up some time ago for some, you could try digging in closed issues and see if some fits your problem. I might also be a good idea to set the verbose flag on and record the logs.

**Reply by damaini** [reply]
Thanks for the suggestion,  I will try a previous EFI release.
How do I set the verbose flag just in case I keep getting the error? (Sorry newbie here)

**Reply by damaini** [reply]
You are a genius!!!!  Got it to work with EFI 0.7.1
I tried 0.7.2.1 and 0.7.2.3, but the install went into a restart loop.
Maybe it has to do with this that I found posted?  I'm running Catalina ver 10.15.7-19H15
**0.7.2 - August 2021 Release
DISCLAIMER: If you're still running macOS Catalina or lower, you have to set the SecureBootModel to j137, otherwise, you wouldn't be able to boot!**

**Reply by simprecicchiani** [reply]
Most likely. Glad it all went well!
